Sources/Rendering/Core/Glyph3DMapper/index.d.ts

```diff
@@ -12,85 +12,101 @@ interface IPrimitiveCount {
 export interface IGlyph3DMapperInitialValues extends IMapperInitialValues {
   orient?: boolean;
   orientationMode?: OrientationModes;
-  orientationArray?: number[];
+  orientationArray?: string;
   scaling?: boolean;
   scaleFactor?: number;
   scaleMode?: ScaleModes;
-  scaleArray?: number[];
+  scaleArray?: string
   matrixArray?: number[];
   normalArray?: number[];
   colorArray?: number[];
```

Comment:
why not change the other array names ?

Reply:
Sorry, I didn't understand your question. Did you mean changing the type for `matrixArray`, etc.? Those are not names but data arrays, hence `number[]`.

Reply:
ohh, I see. I was wrongly assuming that all arrays would behave the same.

my bad.